Dockerfile

```diff
@@ -1,10 +1,13 @@
 FROM centos:centos8
 
-RUN yum install -y git zip unzip jq
+RUN yum install -y git zip unzip jq openssl
 
-RUN curl https://releases.hashicorp.com/terraform/0.12.9/terraform_0.12.9_linux_amd64.zip -o /tmp/terraform_0.12.9_linux_amd64.zip && \
-    unzip /tmp/terraform_0.12.9_linux_amd64.zip -d /usr/local/bin && \
-    rm -rf /tmp/terraform_0.12.9_linux_amd64.zip
+RUN curl http://wpad.internal.ch/websenseproxy_A.cer --output - 2>/dev/null | openssl x509 -inform der -outform pem -out /etc/pki/ca-trust/source/anchors/websense.internal.ch.pem && \
+    update-ca-trust
+
+RUN curl https://releases.hashicorp.com/terraform/0.12.31/terraform_0.12.31_linux_amd64.zip -o /tmp/terraform_0.12.31_linux_amd64.zip && \
```

Comment:
I would probably pull out 0.12.31 as an ARG to make this copy and paste safe

Reply:
Updated in commit e16179f